src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -85,7 +94,7 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
     // Build our image.
     BufferedImage menuImage = null;
     if( menuWidth == 0 || menuHeight == 0 )
-      menuImage = SpriteLibrary.createDefaultBlankSprite(1, 1); // zero-dimensioned images aren't kosher
+      return SpriteLibrary.createDefaultBlankSprite(1, 1); // zero-dimensioned images aren't kosher
```

Comment:
Should this throw an exception instead? Maybe it should.

Reply:
I discovered it kinda does already(?) if you shrink the window enough that it can't even fit one letter in. The window just goes unresponsive.
I ain't sure exactly where that comes from, but I got it to stop doing it for this next commit.

Reply:
I'd still lean towards an exception of some kind. Fail fast, and all that, but up to you.

Reply:
Not gonna go for it. I don't wanna make the game crash, and I don't want to bother catching the exception either.
Linux lets you resize the game window to smaller than the toolbar, leaving no space at all for the text.